### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by NathanBnm**
There were some typos in your app. I tried to fix them. Let me know if you have any questions about it!

### CODE REVIEW

**Comment by torikulhabib** [comment]
Update please

**Comment by NathanBnm** [comment]
@torikulhabib done

File: src/Playlist.vala [hunk 1]
```diff
@@ -152,6 +152,105 @@ namespace niki {
                     }
                 }
                 if (event.button == Gdk.BUTTON_SECONDARY && event.type != Gdk.EventType.2BUTTON_PRESS) {
+                    Idle.add (() => {
```
Comment (torikulhabib) [inline src/Playlist.vala]: for the playlist I've changed the code.
please check againt.

**Comment by NathanBnm** [comment]
@torikulhabib I hope it's okay now

File: src/Playlist.vala [hunk 2]
```diff
@@ -255,7 +255,7 @@ namespace niki {
             int mediatype;
             bool playnow;
             liststore.get (iter, PlaylistColumns.FILENAME, out file_plying, PlaylistColumns.TITLE, out titlename, PlaylistColumns.FILESIZE, out filesize, PlaylistColumns.MEDIATYPE, out mediatype, PlaylistColumns.ALBUMMUSIC, out album, PlaylistColumns.ARTISTMUSIC, out artist, PlaylistColumns.PLAYNOW, out playnow);
-            NikiApp.settings.set_string ("tittle-playing", titlename);
+            NikiApp.settings.set_string ("title-playing", titlename);
```
Comment (torikulhabib) [inline src/Playlist.vala:258]: please change everything related to GLib. Settings
tittle-playing to "title-playing"

Comment (NathanBnm) [reply src/Playlist.vala:258]: Already done!

Comment (torikulhabib) [reply src/Playlist.vala:258]: topbar.vala
mprisplayer
com.github.torikulhabib.niki.gschema.xml

Comment (torikulhabib) [reply src/Playlist.vala:258]: find all related in code "tittle-playing" to "title-playing"

**Comment by NathanBnm** [comment]
Fixed

File: src/PlayerPage.vala [hunk 3]
```diff
@@ -62,8 +62,8 @@ namespace niki {
                     }
                 }
             });
-            NikiApp.settings.changed["activate-subtittle"].connect (() => {
-                playback.subtitle_track = NikiApp.settings.get_boolean ("activate-subtittle")? playback.get_subtitle_track() : -1;
+            NikiApp.settings.changed["activate-subtitle"].connect (() => {
+                playback.subtitle_track = NikiApp.settings.get_boolean ("activate-subtitle")? playback.get_subtitle_track() : -1;
```
Comment (torikulhabib) [inline src/PlayerPage.vala:66]: bottombar.vala
"activate-subtittle" to "activate-subtitle"

**Comment by NathanBnm** [comment]
Fixed too

File: data/icons/actions/com.github.torikulhabib.niki.subtitle-on-symbolic.svg [hunk 4]
```diff
@@ -11,7 +11,7 @@
    version="1.1"
    width="16"
    height="16"
-   sodipodi:docname="com.github.torikulhabib.niki.subtittle-on-symbolic.svg"
+   sodipodi:docname="com.github.torikulhabib.niki.subtitle-on-symbolic.svg"
```
Comment (torikulhabib) [inline data/icons/actions/com.github.torikulhabib.niki.subtitle-on-symbolic.svg:14]: please rename this file...svg

Comment (NathanBnm) [reply data/icons/actions/com.github.torikulhabib.niki.subtitle-on-symbolic.svg:14]: Done already

**Comment by torikulhabib** [comment]
thanks

**Pull request merged**